types/react-native-autocomplete-input/index.d.ts

```diff
@@ -75,15 +69,13 @@ export interface AutocompleteProps<T> extends TextInputProps {
      * function
      * render custom TextInput. All props passed to this function.
      */
-    renderTextInput?(props: TextInputProps): ReactNode;
+    renderTextInput?(props: AutocompleteProps<T>): ReactNode;
 
     /**
-     * string | bool
-     * Set `keyboardShouldPersistTaps` to true if RN version is <= 0.39.
+     * function
+     * render custom result list. Can be used to replace FlatList. All props passed to this function.
      */
-    renderSeparator?: ListViewProps['renderSeparator'] | undefined;
-
-    keyboardShouldPersistTaps?: 'always' | 'handeld' | 'never' | boolean;
+    renderResultList?(props: AutocompleteProps<T>): ReactNode;
```

Comment:
New prop added. For more info, see https://github.com/mrlaessig/react-native-autocomplete-input/pull/230